pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -858,14 +858,40 @@ def get_list_serializer(self, *args, **kwargs):
         kwargs.setdefault("context", self.get_serializer_context)
         return self.list_serializer_class(*args, **kwargs)
 
+    def get_list_queryset(self):
```

Comment:
Is this also beneficial for the detail view? We could just overwrite `get_queryset` at that point.

Reply:
I'm not actually sure, but I can see if it breaks any tests.